editor/js/Sidebar.Geometry.js

```diff
@@ -158,16 +162,21 @@ Sidebar.Geometry = function ( editor ) {
 
 			//
 
-			parameters.clear();
+			var newGeometryType = geometry.type;
+			if (newGeometryType !== lastGeometryType) {
```

Comment:
I made this change because I was running into the following bug...

1. I would select the "Width Segments" UI.Integer to focus it.
2. I would hit UP to nudge the value upward
3. This would change the geometry, which would call the build method (which these changes affect)
4. The build method would *always* recreate the geometry UI, un-focusing the input and making it so that I'd have to select the input to keep nudging